doc/man3/DTLS_set_timer_cb.pod

```diff
@@ -0,0 +1,38 @@
+=pod
+
+=head1 NAME
+
+DTLS_set_timer_cb - Set callback for controlling DTLS timer duration
```

Comment:
The travis failure is because doc-nits is complaining. You need to add the typedef "DTLS_timer_cb" into this list, i.e. "DTLS_set_timer_cb, DTLS_timer_cb"